### DISCUSSION TITLE
Multi Monitor fix/question

### DISCUSSION THREAD

**Discussion by NYPD** [skycommand/AdminScripts#2] [Ideas]
https://github.com/skycommand/AdminScripts/blob/551e29c3a65a3dcd501251aa5a852f5633fa191c/Wallpaper/Find%20current%20wallpaper%20(Windows%208%2C10).ps1#L25

This script works well if there is only one monitor (or you have the same image across multiple monitors), but if you have different images per monitor there is a different TranscodedImageCache _XXX per each one:

![Registry](https://i.imgur.com/a55PZ4Y.png)
(In this case I have 3 monitors)

I have modified the ps1 script to open every single XXX image, which works fine I guess, but I was wondering if you knew how to pass the correct monitor XXX to the script and only open the desktop image on the current monitor one has right clicked.

**Comment by NYPD** [comment]
Wait no, nevermind, these XXX numbers don't match a particular monitor, its just the order of windows transcoding the image

**Comment by NYPD** [comment]
NEVEMIND X2

The XXX numbers do correlate with the monitor numbers seen here: 

![](https://i.imgur.com/J8bIptE.png)

**Comment by skycommand** [comment]
> I was wondering if you knew how to pass the correct monitor XXX to the script and only open the desktop image on the current monitor one has right clicked

I'm afraid I don't.

I think detecting the correct desktop requires a shell extension.

**Comment by adrianium** [comment]
Hello! 

I recently noticed you gave my project a star and even clicked the Watch button. It looks like you find my tool useful. If you don't mind, could you give me your Reddit or other social media account where I can contact you? I'll be happy to improve my tool based on your feedback.

Thanks in advance!

**Reply by skycommand** [reply]
@adrianium Hello. I've seen [your post on Reddit](https://www.reddit.com/r/software/comments/1tvo7kx/i_built_this_free_offline_ai_writing_tool_for/). I'm currently analyzing your tool. Unfortunately, I've ran into problems, namely error `0xc000001d`. I didn't want to leave a comment about it on Reddit until I'm sure because Reddit could be unkind.

**Reply by adrianium** [reply]
@skycommand  Hi! Wow, thank u SO much for ur increadible support and for protecting my Reddit thread from toxivity. U r a lifesaver!

I took this feedback seriously and  fixed the 0xc000001d bug. It should run perfectly smooth on ur hardaware now. 

If u have a free time, could u download and test it? I wonder abt ur oponion and feedback. 

u can download here: https://adrianium.github.io/Scryptian/

Thanks!
